karma.conf.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var karmaWebpack = require('karma-webpack')
+var webpack = require('webpack')
+var path = require('path')
+
+module.exports = function (config) {
+	config.set({
+		frameworks: ['jasmine'],
+    files: [
```

Comment:
hey @JFusco,

can we keep the spacing consistent? i don't mind if we move to tabs but proper indentation makes for clean code. go ahead and edit the editorconfig if you feel we need to change.